src/main/resources/swagger/api-docs.yaml

```diff
@@ -263,6 +263,59 @@ paths:
           description: Internal Error
           schema:
             $ref: '#/definitions/ErrorReport'
+  '/notebooks/{googleProject}/{clusterName}/api/localize':
+    post:
+      summary: Localize files to/from a Jupyter notebook server
+      description: |
+        Sends a command to a Jupyter notebook server to localize files between the server and GCS.
+        This operation will happen asynchronously; output, including any errors, will appear in `localization.log` in the working directory of the Jupyter notebook server.
+      operationId: proxyLocalize
+      tags:
+        - notebooks
+      parameters:
+        - in: path
+          name: googleProject
+          description: googleProject
+          required: true
+          type: string
+        - in: path
+          name: clusterName
+          description: clusterName
+          required: true
+          type: string
+        - in: body
+          description: |
+            JSON object. Keys represent destinations, values represent sources.
+            Paths will be quoted, intermediate local directories made, and then sent to [gsutil cp](https://cloud.google.com/storage/docs/gsutil/commands/cp).
+          name: filesToLocalize
+          required: true
+          schema:
+            type: object
+            description:
+            example:
+              "/localize/to/here" : "gs://some/bucket/object"
```

Comment:
Copying to directories should work, right? E.g. if I had:
```
"/home/rtitle/" : "gs://my-bucket/my-file.tgz"
```
Might be worth noting/having an example in the swagger.

Reply:
Good call. It's in the gsutil docs but I'll add another example too.